main.go

```diff
@@ -172,6 +173,16 @@ func _main(ctx context.Context) error {
 			DBName:       cfg.PGDBName,
 			QueryTimeout: cfg.PGQueryTimeout,
 		}
+		// Replace {netname} with curnet
+		if strings.Contains(dbi.DBName, "{netname}") {
```

Comment:
All .go code must be formatted consistently with `gofmt`.  Practically, this means running `go fmt ./...` at the root of the repo and/or using an IDE that will gofmt code on save.

Reply:
Note the CI (Travis) failure: https://travis-ci.org/decred/dcrdata/builds/464151405?utm_source=github_status&utm_medium=notification